GetStream/stream-chat-swift
# [CIS-1331] Fix rendering unavailable reactions on `ChatMessageReactionAuthorsVC`

### 🔗 Issue Link
CIS-1331

### 🎯 Goal
Fix rendering of unavailable reactions on `ChatMessageReactionAuthorsVC`.

### 🛠 Implementation
The `ChatMessageReactionAuthorsVC` now filters the available reactions like the other components are doing.

### 🎨 Changes
N/A

### 🧪 Testing
- Add a reaction from the React application that the iOS doesn't support (Angry)
- Verify that the reactions don't influence the reaction authors view

### ☑️ Checklist
